lib/init.coffee

```diff
@@ -252,14 +252,17 @@ module.exports =
           messages = @parseSchemaMessages(textEditor, output)
         if messages.length
           for message in messages
-            message.type = 'Error'
-            message.text = message.text + ' (' + schemaUrl + ')'
-            message.filePath = textEditor.getPath()
+            message.severity = 'error'
+            message.excerpt = message.excerpt + ' (' + schemaUrl + ')'
```

Comment:
Ah, I just assumed it was a URL from the name. Note that in v2 messages `message.excerpt` is only rendered as text, links are not clickable.